src/components/ViewItem/ViewItem.js

```diff
@@ -370,15 +370,15 @@ class ViewItem extends React.Component {
             >
               <FormattedMessage id="ui-checkout.checkout.notes" />
             </Button>}
-          { stripes.hasPerm('ui-users.loans.add-patron-info') && !isVirtualUser &&
+          { stripes.hasAnyPerm('ui-users.loans.add-patron-info,ui-users.loans-add-info.create') && !isVirtualUser && // stripes.hasAnyPerm('ui-users.loans.add-patron-info,ui-users.loans-add-info.create')
```

Comment:
Could you please clarify do we need this comments?

Reply:
Thank you for catching it. I removed it now. Maybe I missed something on running yarn lint